src/patternfly/components/FileUpload/file-upload.scss

```diff
@@ -0,0 +1,132 @@
+      opacity: var(--pf-c-file-upload--m-draganddrop--after--Opacity);
+    }
+  }
+}
+
+.pf-c-input-group > .pf-c-form-control {
+  &::placeholder {
+    color: var(--pf-c-file-upload--c-input-group--c-form-control--placeholder--Color);
+  }
+}
+
+.pf-c-file-upload__text-area {
+  position: relative;
+  min-height: var(--pf-c-file-upload__text-area--MinHeight);
+  padding: var(--pf-c-file-upload__text-area--PaddingTop) var(--pf-c-file-upload__text-area--PaddingRight) var(--pf-c-file-upload__text-area--PaddingBottom) var(--pf-c-file-upload__text-area--PaddingLeft);
+  resize: vertical;
+  background-color: var(--pf-c-file-upload__text-area--BackgroundColor);
+  border: var(--pf-c-file-upload__text-area--BorderWidth) solid;
+  border-color: var(--pf-c-file-upload__text-area--BorderTopColor) var(--pf-c-file-upload__text-area--BorderRightColor) var(--pf-c-file-upload__text-area--BorderBottomColor) var(--pf-c-file-upload__text-area--BorderLeftColor);
+
+  &:not(:read-only) {
+    &.pf-m-hover,
+    &:hover {
+      --pf-c-file-upload__text-area--BorderBottomColor: var(--pf-c-file-upload__text-area--hover--BorderBottomColor);
+    }
+
+    &.pf-m-active,
+    &:active {
+      --pf-c-file-upload__text-area--BorderBottomColor: var(--pf-c-file-upload__text-area--active--BorderBottomColor);
+
+      border-bottom-width: var(--pf-c-file-upload__text-area--active--BorderBottomWidth);
+    }
+
+    &.pf-m-focus,
+    &:focus {
+      --pf-c-file-upload__text-area--BorderBottomColor: var(--pf-c-file-upload__text-area--focus--BorderBottomColor);
+
+      border-bottom-width: var(--pf-c-file-upload__text-area--focus--BorderBottomWidth);
+    }
+  }
+
+  &:read-only,
+  &.pf-m-readonly {
+    --pf-c-file-upload__text-area--BackgroundColor: var(--pf-c-file-upload__text-area--readonly--BackgroundColor);
+  }
+
+  &.pf-m-loading {
+    background-color: var(--pf-global--disabled-color--300);
+    border: 0;
+  }
+}
+
+.pf-c-button.pf-m-control {
+  --pf-c-button--PaddingRight: var(--pf-c-file-upload--c-button--m-control--PaddingRight);
+  --pf-c-button--PaddingLeft: var(--pf-c-file-upload--c-button--m-control--PaddingLeft);
+
+  outline-offset: var(--pf-c-file-upload--c-button--m-control--OutlineOffset);
+
+  &:disabled,
+  &.pf-m-disabled {
+    color: initial;
+    background-color: var(--pf-c-file-upload--c-button--m-control--disabled--BackgroundColor);
+
+    &::after {
+      content: "";
+      border: var(--pf-c-file-upload--c-button--m-control--disabled--after--BorderWidth) solid;
+      border-color: var(--pf-c-file-upload--c-button--m-control--disabled--after--BorderTopColor) var(--pf-c-file-upload--c-button--m-control--disabled--after--BorderRightColor) var(--pf-c-file-upload--c-button--m-control--disabled--after--BorderBottomColor) var(--pf-c-file-upload--c-button--m-control--disabled--after--BorderLeftColor);
+      border-radius: initial;
+    }
+  }
+}
+
+.pf-c-spinner {
```

Comment:
The way the styles are written for these components (input group, button, spinner) will apply globally to any use of them in patternfly.